lesson1/dev/sass/_banner.sass

```diff
@@ -0,0 +1,36 @@
+#banner 
+  width: 100%
+  height: 400px
+  position: relative
+  overflow: hidden
+  ol 
+    position: absolute
+    z-index: 200
+    bottom: 10px
+    left: 50%
+    transform: translateX(-50%)
+    li 
+      width: 16px 
+      height: 16px
+      border: 2px solid #fff 
+      border-radius: 100%
+      display: inline-block
+      margin: 0 5px
+      cursor: pointer
+      &.active 
+        background: #fff
+        border: transparent
+  .slider
+    position: absolute
+    top: 0
+    left: 0
+    width: 300%
+    height: 100%
+    display: flex
+    transition: left .5s ease-in-out
```

Comment:
`transition: left .5s ease-in-out`
Tham khảo 
`-webkit-transition: left .5s ease-in-out; /* Safari */
    transition: left .5s ease-in-out;`
Sửa lại toàn bộ những phần tương tự